### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by sluzky**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/sluzky/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
+`['!fixed', '!resolved', '!completed']`
+- The issue has been resolved.
+
+To add a flag simply leave a comment with your desired flag. For
```
Comment (sluzky) [inline ReadMe.md]: Weird wording !fix

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
```
Comment (sluzky) [inline ReadMe.md]: Oh god!

Comment (sluzky) [inline ReadMe.md]: !fix

File: ReadMe.md [hunk 3]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (sluzky) [inline ReadMe.md]: WOW!

Comment (sluzky) [inline ReadMe.md]: !fix

File: ReadMe.md (hunk 2 above)
Comment (sluzky) [inline ReadMe.md]: fixed this !fixed

File: ReadMe.md (hunk 3 above)
Comment (sluzky) [inline ReadMe.md]: !completed

File: ReadMe.md [hunk 4]
```diff
@@ -1 +1,41 @@
...
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
+`['!fixed', '!resolved', '!completed']`
+- The issue has been resolved.
+
+To add a flag simply leave a comment with your desired flag. For
```
Comment (sluzky) [inline ReadMe.md]: !resolved

File: ReadMe.md (hunk 3 above)
Comment (sluzky) [inline ReadMe.md]: fixed !fixed

File: ReadMe.md (hunk 4 above)
Comment (sluzky) [inline ReadMe.md]: !fix

Comment (sluzky) [inline ReadMe.md]: !fixed

**Pull request merged**